test/evergreen.yml

```diff
@@ -4901,6 +4904,72 @@ buildvariants:
       execution_tasks:
       - ".wt2853-perf"
 
+- name: ubuntu2004-perf-tests-accuracy
+  display_name: Ubuntu 20.04 Performance tests (improved accuracy)
+  batchtime: 1440 # 1 day
+  run_on:
+    - ubuntu2004-medium
+  expansions:
+    test_env_vars:
+      WT_TOPDIR=$(git rev-parse --show-toplevel)
+      WT_BUILDDIR=$WT_TOPDIR/cmake_build
+      LD_LIBRARY_PATH=$WT_BUILDDIR:$WT_BUILDDIR/test/utility/
+    CC_OPTIMIZE_LEVEL: -DCC_OPTIMIZE_LEVEL=-O3
+    HAVE_DIAGNOSTIC: -DHAVE_DIAGNOSTIC=0
+    CMAKE_TOOLCHAIN_FILE: -DCMAKE_TOOLCHAIN_FILE=../cmake/toolchains/mongodbtoolchain_v3_gcc.cmake
+    CMAKE_INSTALL_PREFIX: -DCMAKE_INSTALL_PREFIX=$(pwd)/cmake_build/LOCAL_INSTALL
+    python_binary: '/opt/mongodbtoolchain/v3/bin/python3'
+    pip3_binary: '/opt/mongodbtoolchain/v3/bin/pip3'
+    virtualenv_binary: '/opt/mongodbtoolchain/v3/bin/virtualenv'
+    smp_command: -j $(echo "`grep -c ^processor /proc/cpuinfo` * 2" | bc)
+    cmake_generator: Ninja
+    make_command: ninja
+    improved_accuracy: '--opts_flag'
+    max_runs_acc: 5
+  tasks:
+    - name: compile
+    - name: ".btree-perf"
+    # Disable LSM perf tests till the support for LSM is restored.
+    # - name: ".lsm-perf"
+    - name: ".stress-perf"
+    - name: ".checkpoint-perf"
+    - name: ".evict-perf"
+    - name: ".log-perf"
+    - name: ".long-perf"
+    - name: ".oplog-perf"
+    - name: many-dhandle-stress
```

Comment:
minor: We can remove `many-dhandles-stress` and `wt2853-perf` from this variant as they don't use wtperf so our `improved_accuracy flag` won't apply.

Reply:
Good catch, thanks! https://github.com/wiredtiger/wiredtiger/pull/8469/commits/bbff47e0ab9018d913aa175d70dba79d91e3a0f2